devlab/tests/generate_load_real_env.py

```diff
@@ -127,115 +390,325 @@ def emulate_vm_states(self):
+                        time.sleep(3)
+                        logging.info(">>> Security Group: : %s Deleted !!" % sg['name'])
             except Exception as e:
-                print "Security group %s failed to delete: %s" % (sg['name'],
-                                                                  repr(e))
+                logging.warning("Security group %s failed to delete: %s" % (sg['name'],
+                                                                  repr(e)))
+        logging.info('>>> Cleaning Volume Snapshots:')
         snapshots = self.config.cinder_snapshots
         for snapshot in snapshots:
+            snapshot['display_name'] = self.update_name_with_prefix(snapshot['display_name'])
             try:
+                time.sleep(1)
                 self.cinderclient.volume_snapshots.delete(
                     self.get_volume_snapshot_id(snapshot['display_name']))
+                logging.info(">>> Volume Snapshot: : %s Deleted !!" % snapshot['display_name'])
             except Exception as e:
-                print "Snapshot %s failed to delete: %s" % (
-                    snapshot['display_name'], repr(e))
+                logging.warning("Snapshot %s failed to delete: %s" % (
+                    snapshot['display_name'], repr(e)))
+        logging.info('>>> Cleaning Volumes:')
         volumes = self.config.cinder_volumes
         for volume in volumes:
+            volume['name'] = self.update_name_with_prefix(volume['name'])
             try:
-                self.cinderclient.volumes.delete(
-                    self.get_volume_id(volume['name']))
+                time.sleep(1)
+                self.cinderclient.volumes.delete(self.cinderclient.volumes.get(
+                                           self.get_volume_id(volume['name'])))
+                logging.info(">>> Volume: : %s Deleted !!" % volume['name'])
             except Exception as e:
-                print "Volume %s failed to delete: %s" % (volume['name'],
-                                                          repr(e))
+                logging.warning("Volume %s failed to delete: %s" % (volume['name'],
+                                                          repr(e)))
+
+        logging.info('>>> Cleaning Flavors:')
+        self.clean_flavors()
+        time.sleep(2)
+        logging.info('>>> Cleaning Tenant:')
+        self.clean_tenant()
 
 
 if __name__ == '__main__':
-    preqs = Prerequisites(config=real_env_conf)
     parser = argparse.ArgumentParser(
         description='Script to generate load for Openstack and delete '
-                    'generated objects')
+                    'generated objects',formatter_class=RawTextHelpFormatter)
     parser.add_argument('--clean', action='store_true',
                         help='clean objects described in real_env_conf.ini')
+
+    parser.add_argument('--option', type=int,
+                        help="creates single resources\n"
```

Comment:
re-use the RESOURCE_CREATE_MAP dictionary to create the help message, now we must support this dictionary in two places.